pkgs/by-name/ro/rosa/package.nix

```
@@ -2,31 +2,25 @@
 
 buildGoModule rec {
   pname = "rosa";
-  version = "1.2.46";
+  version = "1.2.48";
 
   src = fetchFromGitHub {
     owner = "openshift";
     repo = "rosa";
     rev = "v${version}";
-    hash = "sha256-XRoHapuH0MJNrtu+Rk/yxJqeqjNIbdGYqDB84q05rdA=";
+    hash = "sha256-qJKzJrCZKhaqoRxloTUqaRsR4/X/hoMxmDQCTNccTqk=";
   };
   vendorHash = null;
 
   ldflags = [ "-s" "-w" ];
 
   __darwinAllowLocalNetworking = true;
 
-  postPatch = ''
-    # e2e tests require network access
-    rm -r tests/e2e
-  '';
+  # skip e2e tests package
+  excludedPackages = [ "tests/e2e" ];
 
-  preCheck = ''
-    # Workaround for cmd/list/rhRegion/cmd_test.go:39
-    #   Failed to get OCM regions: Can't retrieve shards: Get "https://api.stage.openshift.com/static/ocm-shards.json": dial tcp: lookup api.stage.openshift.com on [::1]:53: read udp [::1]:55793->[::1]:53: read: connection refused
-    substituteInPlace "cmd/list/rhRegion/cmd_test.go" \
-      --replace-fail "TestRhRegionCommand" "SkipRhRegionCommand"
-  '';
+  # skip tests that require network access
+  checkFlags = [ "-skip=^TestCluster$" ];
```

```suggestion
  checkFlags =
    let
      skippedTests = [
        "TestRhRegionCommand"
        "TestCluster"
      ];
    in
    [ "-skip=^${lib.concatStringsSep "$|^" skippedTests}$" ];
```

The test previously patched out [still fails](https://logs.ofborg.org/?key=nixos/nixpkgs.359440&attempt_id=1ba2ef0e-dae8-496d-966e-d28e394ab8b9) here. Adding this fixed building on x86_64-linux

---

I didn't have a proper linux/nixos environment to test in and that test does not fail on darwin (with sandboxing). In any case, I have re-added that test to the list and now I have a linux environment to test builds in. Thank you!